packages/cli/src/index.ts

```diff
@@ -85,10 +85,10 @@ function printHelpInformation(
 }
 
 function printUnknownCommand(cmdName: string) {
-  const suggestion = didYouMean(
-    cmdName,
-    commander.commands.map(cmd => cmd._name),
-  );
+  const availableCommands = commander.commands.map(cmd => cmd._name);
```

Comment:
We don't need this call. We can simply do `const suggestion = commander.commands.find(cmd => level(cmd._name, cmdName) < 3);`

Reply:
```js
const suggestion = commander.commands.find(cmd => leven(cmd._name, cmdName) < 3)._name;
```
That would be hard to read. I would say grabbing command names from the entire `commander.commands` array would be the way to go.